tests/unittest/conftest.py

```diff
@@ -7,103 +7,17 @@
 import sys
 from unittest import mock
 
-from everett.manager import ConfigManager, ConfigDictEnv, ConfigOSEnv
-from falcon.request import Request
-from falcon.testing.helpers import create_environ
-from falcon.testing.client import TestClient
-import markus
 from markus.testing import MetricsMock
 import pytest
 
 
-# Add repository root so we can import antenna and testlib.
+# Add repository root so we can import testlib.
 REPO_ROOT = Path(__file__).parent.parent.parent
 sys.path.insert(0, str(REPO_ROOT))
```

Comment:
Doesn't this and the equivalent code in `tests/conftest.py` cause `sys.path.insert` to insert the same path twice? `antenna` and `testlib` modules are both in the same directory.

Reply:
this won't be a concern because i'm going to put the new tests in tests/unittest and revert all changes to this file, but:

yes it does, but it's also already in the path when tests are run via `bin/run_tests.sh` because of `export PYTHONPATH=/app/:${PYTHONPATH:-}`, and leaving it in both places means they aren't dependent on eachother to continue functioning.